pkg/config/samples/default.dusk.toml

```diff
@@ -22,6 +22,8 @@ timeoutsendstaketx = 5
 timeoutgetmempooltxs = 3
 timeoutgetroundresults = 5
 timeoutbrokergetcandidate = 2
+timeoutreadwrite = 60
+timeoutkeepalivetime = 30
```

Comment:
It should be good to clarify here as a comment that `timeoutkeepalivetime`  must be always smaller than `timeoutreadwrite`, otherwise the node will disconnect due to `read timeout error`

Reply:
sure